modules/p2/ding_entity_rating/ding_entity_rating.module

```diff
@@ -365,7 +365,7 @@ function ding_entity_rating_theme($existing, $type, $theme, $path) {
  * Theme function for theme_ding_entity_rating_thank_you.
  */
 function theme_ding_entity_rating_thank_you($variables) {
-  return '<div class="ding-entity-rating-respons">' . $variables['response'] . '</div>';
+  return '<div class="ding-entity-rating-respons">' . t('@ding_entity_rating_thank_you', ['@ding_entity_rating_thank_you' => $variables['response']]) . '</div>';
```

Comment:
Make sure that in translate interface wee actual English text, not the placeholder.